challenge.py

```diff
@@ -0,0 +1,117 @@
+import os
+import re
+import numpy as np
+
+List_calls = "calls.txt"
+List_files = "list_files.txt"
+result_stat = "stat.txt"
+flag_file = 1
+flag_calls = 1
+def result(filename):
+  
+  #local variable declaration
+  deleted = 0 
+  added = 0 
+  region = 0 
+  
+  #Regular expression declaration 
+  regex_FileName = '^diff --git a.+[ ]b/(.+?)\n'
+  regex_Del = "^-(?!--).*"
+  regex_Add = "^\+(?!\+\+).*"
```

Comment:
Lookaheads are rather expensive, there should be a faster way to check this

Reply:
For the added and deleted lines, I think it should be ok, because if I detect only '-' or '+' it will catch lines starting by '---' or '+++'. Do you have suggestion ?
And for file name, I choose to detect the b/ file (the new one), because the file could be renamed and the new name will be on the b/ side.